packages/manager/modules/navbar/src/notifications-menu/translations/Messages_fr_FR.json

```diff
@@ -1,3 +1,4 @@
 {
-  "navbar_notification_title": "Vos notifications"
+  "navbar_notification_title": "Vos notifications",
+  "navbar_notification_new": "nouvelle"
```

Comment:
```suggestion
  "navbar_notification_new": "New
```
;)

Reply:
New is New in French?

Reply:
No but the mockup includes `new` and `nouvelle` would just be ugly imo :)